core/src/main/java/eu/kevin/core/enums/KevinCountry.kt

```diff
@@ -31,8 +31,6 @@ enum class KevinCountry(val iso: String) {
     SLOVAKIA("sk");
 
     companion object {
-        fun parse(iso: String?): KevinCountry? {
-            return values().firstOrNull { it.iso.lowercase() == iso }
-        }
+        fun parse(iso: String?) = values().firstOrNull { it.iso.equals(iso, ignoreCase = true) }
```

Comment:
Hey, thank you for the PR! 
Please keep our code conventions, that means type should be visible in the code.

Reply:
Sure